core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.module.ts

```diff
@@ -75,6 +78,10 @@ import { MergeSkillModalComponent } from 'components/skill-selector/merge-skill-
       useFactory: platformFeatureInitFactory,
       deps: [PlatformFeatureService],
       multi: true
+    },
+    {
+      provide: '$scope',
+      useExisting: '$rootScope'
```

Comment:
Let's not do this. It's a bad pattern. This solves our problem in the short run but will create a long-term problem. After this is merged, maybe there is some migration we do that is using this, but we don't know because we won't get any errors. After the migration of directives is done, we remove this and suddenly run into a ton of errors. It might look harmless but this is very dangerous. This is what happened with UpgradedService.ts file.

Reply:
fixed.